lib/DDG/Spice/GithubStatus.pm

```diff
@@ -5,7 +5,12 @@ use strict;
 use DDG::Spice;
 use Text::Trim;
 
-triggers startend => 'github';
+triggers start => "github status", "github system status";
+
+handle remainder => sub {
```

Comment:
We already have a `handle` function below. That one should be replaced with your new one 👍